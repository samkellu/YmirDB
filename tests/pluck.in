pluck a 1
set a
pluck a
set a 123 22
pluck a 5
get a
pluck a 1
set b a 22 45 56
pluck b 1
get b
get a
bye
