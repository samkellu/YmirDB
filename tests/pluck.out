> no such key

> ok

> > ok

> index out of range

> [123 22]

> 123

> ok

> a

> [22 45 56]

> [22]

> bye
